content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -21,6 +21,18 @@ deprecated API versions to newer and more stable API versions.
 ## Removed APIs by release
 
 
+### v1.27
+
+The **v1.27** release will stop serving the following deprecated API versions:
+
+#### HorizontalPodAutoscaler {#horizontalpodautoscaler-v127}
+
+The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler will no longer be served in v1.27.
+
+* Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
+* All existing persisted objects are accessible via the new API
+* No notable changes
```

Comment:
I think some field names have been renamed(?)

Reply:
BTW, https://github.com/kubernetes/enhancements/tree/master/keps/sig-autoscaling/2702-graduate-hpa-api-to-GA#renames is my basis for suspecting this.

Reply:
@josephburnett @gjtempleton , please confirm if there are field names renamed

Reply:
I believe this change ended up not happening due to the extra work that would have been required, see discussion here: https://github.com/kubernetes/kubernetes/pull/102534/files#r662447798